packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -212,7 +212,8 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
         onRenderAriaDescription(props, this._onRenderAriaDescription),
         onRenderChildren(props, this._onRenderChildren),
         !this._isSplitButton && (menuProps || menuIconName || menuIconProps || this.props.onRenderMenuIcon) && onRenderMenuIcon(this.props, this._onRenderMenuIcon),
-        this.state.menuProps && onRenderMenu(menuProps, this._onRenderMenu)
+        // If button is disabled, do not render its menu.
+        this.state.menuProps && !this.props.disabled && onRenderMenu(menuProps, this._onRenderMenu)
```

Comment:
Can we instead make the menuProps not get set instead when the component is disabled? Is the click handler firing for the chevron even though the component is disabled? With this approach, the isExpanded method will return the incorrect result for the disabled state because it uses this.state.menuProps